rfcs/0018-glides-trainsheet-events.md

```diff
@@ -0,0 +1,403 @@
+- Feature Name: `glides-trainsheet-events`
+- Start Date: 2023-01-20
+- RFC PR: [mbta/technology-docs#18](https://github.com/mbta/technology-docs/pull/18)
+- Status: Proposed
+
+# Summary
+Glides will provide high-level trainsheet information as CloudEvents in a Kinesis stream
+
+# Motivation
+As part of digitizing the currently paper-based trainsheets that define operational conditions on the Green Line, Glides will be gathering information about consist assignments, dropped/added trips, and departure time adjustments from its users: pull-out inspectors at selected terminal stations. This data can serve to improve the rider-facing predictions produced by RTR, especially for departure times at a terminals, which currently do not have departure predictions at all.
+
+The goal of this RFC is to provide an interface which represents all data currently (2023H1) entered into Glides. While some additional flexibility is included, features outside the current scope of Glides are also outside the scope of these events and this RFC.
```

Comment:
@hackajesse @skyqrose @boringcactus added this to represent the overall goal of this RFC, which hopefully clarifies some of the decisions.